scripts-available/CDB_Groups_API.sql

```diff
@@ -0,0 +1,57 @@
+-- Requires configuration parameter. Example: SELECT cartodb.CDB_Conf_SetConf('groups_api', '{ "host": "127.0.0.1", "port": 3000, "timeout": 10, "username": "superadmin", "password": "monkey" }');
+
+-- Sends the create group request
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Group_CreateGroup_API(database_name text, group_name text, group_role text)
+    RETURNS VOID AS
+$$
+    import httplib
```

Comment:
We should try to cache module loading when possible: http://www.postgresql.org/docs/9.1/static/plpython-sharing.html